### PR TITLE
Slides update and fixes for BTCPrague workshop

### PR DESCRIPTION
**Pull request by GitGab19**
TODO:

- [x] Add supporters slide
- [x] Add Q&A + conclusion slide
- [x] Cleanup commented markdown

### CODE REVIEW

**Comment by plebhash** [comment]
@GitGab19 FYI I switched the JDS coinbase `fixme` from `P2PK` to `P2WPKH` on the config file:

https://github.com/stratum-mining/stratum/commit/0094b7997422dc39dbe7cb9f03280cef06f856ed

It allows to visualize the blocks mined per address (instead of per pubkey).

For example, I mined some blocks on `testnet4`:
- `P2PK`: https://mempool.space/testnet4/address/03320029aa5f314eacf81ef6d3c0efd43bdec26dc4bd822c7b8721b3279d84269c
- `P2WPKH`: https://mempool.space/testnet4/address/tb1qujdhsy5zsstq9ydmvaxsh6uthpvakkn3amln4t

no changes are needed on the slides, just leaving this comment here so you're aware.

also, I haven't fixed the local mempool.space deployment on my machine yet, not sure if I will have time.

so maybe this change will be meaningless, but if we end up having mempool.space, it will be good for the audience to see the blocks they mined.

**Comment by Sjors** [comment]
I started running through the workshop.

I'll add an Intel macOS binary to the latest release.

Maybe we should encourage people to perform the `nix-shell` instruction _before_ they go to the workshop (http://75.119.150.111:8888/html/sv2-workshop.html#27), because it might blow up the wifi (168.76 MiB download for me).

On macOS 14.5 running `tproxy` complains:

```
tproxy
error: failed to parse manifest at `/Users/sjors/dev/stratum/roles/translator/Cargo.toml`

Caused by:
  feature `edition2021` is required

  consider adding `cargo-features = ["edition2021"]` to the manifest
```

Couple nits on the slides, feel free to ignore:

* page 6 Template Provider: "custom bitcoind node": maybe mention that the goal is to get that merged eventually

I also can't copy-paste anything from the PDF. Or when I do it turns into gibberish: "rehtohcae ot pihsnoitaler ni maertspu ro maertsnwod..."

File: md/sv2-workshop.md [hunk 1]
```diff
@@ -415,13 +322,30 @@ cargo run -- -c tproxy-config-btcpp-workshop.toml
...
-- buildable source: `https://github.com/pooler/cpuminer`
+- downloadable binaries: [`https://sourceforge.net/projects/cpuminer/files/`](https://sourceforge.net/projects/cpuminer/files/)
+- buildable source: [`https://github.com/pooler/cpuminer`](https://github.com/pooler/cpuminer)
 - nix: `nix-shell -p cpuminer`
 
 To start mining:
 
 ```
-minerd -a sha256d -o stratum+tcp://localhost:34255 -q -D -P
+./minerd -a sha256d -o stratum+tcp://localhost:34255 -q -D -P
```
Comment (Sjors) [inline md/sv2-workshop.md:332]: Don't add `./` in the nix shell.

**Comment by GitGab19** [comment]
> I started running through the workshop.
> 
> I'll add an Intel macOS binary to the latest release.
> 
> Maybe we should encourage people to perform the `nix-shell` instruction _before_ they go to the workshop (http://75.119.150.111:8888/html/sv2-workshop.html#27), because it might blow up the wifi (168.76 MiB download for me).
> 
> On macOS 14.5 running `tproxy` complains:
> 
> ```
> tproxy
> error: failed to parse manifest at `/Users/sjors/dev/stratum/roles/translator/Cargo.toml`
> 
> Caused by:
>   feature `edition2021` is required
> 
>   consider adding `cargo-features = ["edition2021"]` to the manifest
> ```
> 
> Couple nits on the slides, feel free to ignore:
> 
>     * page 6 Template Provider: "custom bitcoind node": maybe mention that the goal is to get that merged eventually
> 
> 
> I also can't copy-paste anything from the PDF. Or when I do it turns into gibberish: "rehtohcae ot pihsnoitaler ni maertspu ro maertsnwod..."

I can't reproduce this error on my macOS, it just works. Is your mac ARM?

**Comment by Sjors** [comment]
Intel

**Pull request merged**